docs/commands/bootstrap.mdx

```diff
@@ -93,7 +93,12 @@ melos bootstrap --diff="main"
 
 ## Bootstrap flags
 
-Melos bootstrap command supports a few different flags that can be defined in
+
+- `--enforce-lockfile` flag to enforce versions from `.lock` files
+    - Ensure .lock files exist, as failure may occur if they're not checked in.
```

Comment:
I think it should be consistent with the `--no-example` flag, but this place is better for the flags, maybe you could move down the `--no-example` docs to this list (instead of moving this one up)? The `--no-example` flag is here:
https://github.com/invertase/melos/pull/600/files#diff-7e3f0cab47656b0eb3a4edbd45ab0cd7a8bb1f96d3d15c8103dee8c863940382R22-R28